package/MDAnalysis/auxiliary/base.py

```diff
@@ -545,7 +546,35 @@ def represent_ts_as(self, new):
     def __del__(self):
         self.close()
 
-    
+    def get_description(self):
+        """Get the args/kwargs necessary for dupicating the AuxReader
+
+        The resulting description may be passed to 'add_auxiliary' to 
+        reload an auxiliary into a trajectory.
+
+        Returns
+        -------
+        dict
+            Of args/kwargs that can be used to recreate the AuxReader
+        """
+        description = {key.lstrip('_'): val 
+                       for key, val in self.__dict__.items()
+                       if key in ['constant_dt', 'cutoff', '_time_col', 
+                                  '_initial_time', '_represent_ts_as', '_dt',
+                                  '_data_cols', 'auxname']}
+        description['format'] = self.format
+        description['auxdata'] = self._data_input
+        return description
+
+    def __eq__(self, other):
+        for attr in ['_represent_ts_as', 'cutoff', '_dt', '_initial_time',
```

Comment:
Same as above, I would store the list in a separate variable to avoir the very long for statement.